.github/workflows/image-publish-on-push-to-branch.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     permissions:
       contents: read
   nest_test_and_sonarcloud:
-    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-nest-test-sonarcloud.yaml@1.3.1
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-nest-test-sonarcloud.yaml@DBP-332-sonarcloud
```

Comment:
Change when https://github.com/dBildungsplattform/dbp-github-workflows/pull/6 is merged and version tag added.